### DISCUSSION TITLE
Ignore any unspecified transition by default?

### DISCUSSION THREAD

**Discussion by q-0q** [dotnet-state-machine/stateless#594] [Q&A]
Hi all, is it possible to configure the StateMachine to assume that **any** unspecified transition should be ignored when calling `Fire`? 

If this is your desired behavior and you have many states and triggers, configuring every state / trigger pair with the `Ignore` feature is very tedious. I'm wondering if there's some kind of StateMachine-level configuration flag I can set to do this. Thanks.

**Comment by q-0q** [comment]
Worked around by using a universal superstate that all states inherit from; and ignoring all triggers for that superstate.

**Comment by crozone** [comment]
From the README: https://github.com/dotnet-state-machine/stateless/blob/dev/README.md#ignored-transitions-and-reentrant-states

You can simply use:

```csharp
stateMachine.OnUnhandledTrigger((state, trigger) => { });
```
